uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/POMServiceImpl.java

```diff
@@ -97,6 +111,25 @@ private MavenRepository getRepository(final String baseURL) {
         return mavenRepository;
     }
 
+    private boolean write(Model model, org.uberfire.java.nio.file.Path pathToPOMXML, IOService ioService) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        try {
+            writer.write(baos, model);
+            ioService.write(pathToPOMXML, new String(baos.toByteArray(), StandardCharsets.UTF_8));
+            return true;
+        } catch (Exception e) {
+            return false;
```

Comment:
It might be good to at least log the exception so users know that pom migration failed.

Reply:
Logged